docs/modules/ROOT/pages/browser-sso.adoc

```diff
@@ -0,0 +1,112 @@
+[[browser-sso]]
+= Neo4j Browser Single Sign-On (SSO)
+:description: This section describes SSO for Browser.
+
+Browser provides basic support for some Single Sign On (SSO) providers under specific Enterprise environment configurations.
+Contact your Neo4j representative for additional assistance.
+Your organization needs an existing compatible SSO provider (local or external) already configured.
+
+Supported providers at this time include OpenID Connect (OIDC) OAuth 2.0 providers Okta, KeyCloak, and Microsoft Azure AD.
+SSO has been tested on Neo4j 4.2.x and 4.3.x deployments.
+Browser supports two authorization flows:
+* Authorization Code flow with PKCE
+* Implicit flow
+
+[IMPORTANT]
+====
+It is strongly advised to use PKCE to ensure security.
+Further information about OpenID Connect and OAuth can be found at https://openid.net/connect/.
+====
+
+In addition to having an SSO provider already configured at your organization, configuring SSO for Browser and Neo4j requires a separate OAuth plugin, available from your Neo4j representative.
+With these prerequisites in place, two additional steps are required:
+
+. The https://neo4j.com/docs/operations-manual/current/configuration/neo4j-conf/#neo4j-conf[`neo4j.conf`] file should be updated with the following information (be sure to avoid duplicate entries in the `neo4j.conf` file):
++
+[source, properties]
+----
+dbms.security.auth_enabled=true
+dbms.security.authentication_providers=native,plugin-org.neo4j.auth.openid.OpenIdPlugin
+dbms.security.authorization_providers=native,plugin-org.neo4j.auth.openid.OpenIdPlugin
+plugins.auth.openid.configuration=https://login.org.com/.well-known/openid-configuration
+plugins.auth.openid.group_to_role_mapping=reader=reader;editor=editor;publisher=publisher;architect=architect;admin=admin
+plugins.auth.openid.claim.groups=groups
+plugins.auth.openid.claim.principal=preferred_username
+----
++
+Optionally, you may set extra logging for the OAuth2 plugin with these settings in the `neo4j.conf` file.
+The logs are found in the `neo4j.log` file.
++
+[source, properties]
+----
+dbms.jvm.additional=-Dorg.apache.commons.logging.Log=org.apache.commons.logging.impl.SimpleLog
+dbms.jvm.additional=-Dorg.apache.commons.logging.simplelog.showdatetime=true
+dbms.jvm.additional=-Dorg.apache.commons.logging.simplelog.log.org.apache.http=DEBUG
+----
+
+Browser needs to be aware of the identity providers available for use.
+This is done by specifying a URL parameter discoveryURL that specifies a URL to a json file containing the SSO providers.
+Example for Browser: `http://<browser-server-host>:<http-port>?discoveryURL=http://webhost.com/public/discovery.json`.
```

Comment:
@OskarDamkjaer can you please confirm this example url is correct?

Reply:
Looks like correct example to me!

Reply:
We could mention that it's planned to be picked up from the server in the future

Reply:
What is planned to be picked up from the server?

Reply:
There's a built in discovery.json as well when the browser is hosted by neo4j, it currently only has `bolt` and some other fields but in the future it'll be configurable to have SSOProviders as well. But we can wait with documenting that until it's available in 4.4